Sources/BraintreeApplePay/Public/BraintreeApplePay/BTApplePayCardNonce.h

```diff
@@ -16,10 +16,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, readonly, strong) BTBinData *binData;
 
+/**
+ The last two digits of the card associated with this nonce, if available.
```

Comment:
```suggestion
 The last two digits of the Device Primary Account Number (DPAN) associated with this nonce, if available.
```

Reply:
We removed this property entirely. We originally thought DPAN was the same as PAN.